device/thunder_ripple_sdk/src/processors/thunder_persistent_store.rs

```diff
@@ -118,11 +118,11 @@ impl ThunderStorageRequestProcessor {
             .get_thunder_client()
             .call(DeviceCallRequest {
                 method: thunder_method,
-                params: params,
+                params,
             })
             .await;
         if response.message.get("success").is_none()
-            || response.message["success"].as_bool().unwrap() == false
+            || !response.message["success"].as_bool().unwrap()
```

Comment:
Since we are touching this part, can we replace unwrap with unwrap_or_default() which is safer.